### PR TITLE
Write announcement for paris rustfest

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

**Comment by Hoverbear** [comment]
👍 Looks good.

File: _posts/2018-02-05-lets-step-in-paris.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: "How about a french adventure in Paris?"
+socialImageSrc: "/assets/paris/paris-panorama.jpg"
+authors:
+  - GuillaumeGomez
```
Comment (GuillaumeGomez) [inline _posts/2018-02-05-lets-step-in-paris.md]: @Hoverbear: You forgot to add yourself here.

File: _posts/2018-02-05-lets-step-in-paris.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+---
+
+How about enjoying Rust in the french capital? After the adventures in Berlin, Київ and Zürich, it's time to put our luggages in Paris, France! We hope you'll join us on **May 26th and October 27th, 2018** at the incredible [**Espace Saint-Martin**](http://espacesaintmartin.com/fr/contact/) in the center of Paris.
+
+[![Paris the city](assets/paris/paris-panorama.jpg)](https://www.flickr.com/photos/)
```
Comment (GuillaumeGomez) [inline _posts/2018-02-05-lets-step-in-paris.md]: We need two pictures to put in here (a small one and a big one).

Comment (Hoverbear) [reply _posts/2018-02-05-lets-step-in-paris.md]: Do you have any nice shots? Cell phone shots are totally okay.

Comment (GuillaumeGomez) [reply _posts/2018-02-05-lets-step-in-paris.md]: Funny thing: I've been living in paris for years and I have no outside pics. But I think it's pretty easy to find considering it's Paris...

**Pull request merged**